<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingdialog</class>
 <widget class="QDialog" name="settingdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="musicbutton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="musiclabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>轻音乐:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="songNumLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0首</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filebutton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background:transparent;
	border:none;
}
QPushButton:hover{
	background-color:rgba(153, 153, 153, 60);
	border-radius:4px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7,2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="musicSlider">
       <property name="styleSheet">
        <string notr="true"> QSlider::groove:horizontal {
     height: 4px; 
     background:rgba(135, 135, 135, 50)
 }

 QSlider::handle:horizontal {
	 background:rgb(207, 68, 53);
     width: 16px;
     margin-top: -5px; 
     margin-bottom:-5px;
     border-radius: 5px;
 }
QSlider::sub-page:horizontal{
background:rgba(255, 85, 65, 100);
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bmusicbutton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bmusiclabel">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>白噪音:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bmusicBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filebutton2">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background:transparent;
	border:none;
}
QPushButton:hover{
	background-color:rgba(153, 153, 153, 60);
	border-radius:4px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,7,2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="bmusicSlider">
       <property name="styleSheet">
        <string notr="true"> QSlider::groove:horizontal {
     height: 4px; 
     background:rgba(135, 135, 135, 50)
 }

 QSlider::handle:horizontal {
	 background:rgb(207, 68, 53);
     width: 16px;
     margin-top: -5px; 
     margin-bottom:-5px;
     border-radius: 5px;
 }
QSlider::sub-page:horizontal{
background:rgba(255, 85, 65, 100);
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>工作时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="worktBox">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>休息时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sleeptBox">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
